notebooks/llm-agent-mcp/llm-agent-mcp.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
<p>7860 and for linux too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2970/discussion/#comment-2137909054'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>